app/workers/syncsIn.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use Utopia\App;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Queue;
+use Utopia\Queue\Message;
+
+require_once __DIR__ . '/../init.php';
+
+define("CURRENT_REGION", App::getEnv('_APP_REGION', 'nyc1'));
+
+/**
+ * @return RedisCache
+ */
+function getCache(): RedisCache
+{
+    global $register;
+    return new RedisCache($register->get('cache'));
+}
+
+$connection = new Queue\Connection\Redis('redis');
```

Comment:
As mentioned on init.php. This is not the way to create a connection. You have to sync branches an imitate what we do for DB connections and instances.